packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/UpgradeNowSilver/template.success.tsx

```diff
@@ -48,25 +49,17 @@ const RowItemSubTitle = styled(Text)`
   flex: 1;
 `
 
-const RowItemTitleWhite = styled(RowItemTitle)`
-  color: ${(props) => props.theme.white};
-`
-
-const RowItemSubTitleWhite = styled(RowItemSubTitle)`
-  color: rgba(255, 255, 255, 0.72);
-`
-
 const StatusCartridge = styled(GreyCartridge)`
   display: flex;
   flex-direction: row;
   > span {
     padding: 3px 8px;
   }
 `
-const StatusCartridgeBlue = styled(BlueCartridge)`
+const StatusCartridgeSuccess = styled(SuccessCartridge)`
```

Comment:
No need to change this now, but for future reference you could use the constellation `<Tag />` component to remove the need for this. https://blockchain.github.io/constellation/?path=/story/primitives-tag--tag&args=variant:success